src/nupic/algorithms/sdr_classifier_factory.py

```diff
@@ -0,0 +1,61 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""Module providing a factory for instantiating a SDR classifier."""
+
+from nupic.algorithms.sdr_classifier import SDRClassifier
+from nupic.support.configuration import Configuration
+
+
+
+class SDRClassifierFactory(object):
+  """Factory for instantiating SDR classifiers."""
+
+
+  @staticmethod
+  def create(*args, **kwargs):
```

Comment:
In the absence of well defined args, we should have a complete docstring with details of what is expected and required.